core/src/martian/schema.cljc

```diff
@@ -130,7 +130,8 @@
               [(schema-type definitions (assoc (:items schema) :required true))]
 
               (= "object" type)
-              (schemas-for-parameters definitions (map (fn [[name p]] (assoc p :name name)) properties))
+              (cond-> (schemas-for-parameters definitions (map (fn [[name p]] (assoc p :name name)) properties))
+                (or additionalProperties (= additionalProperties {})) (assoc (s/optional-key s/Any) s/Any))
```

Comment:
Does this `or` do anything? Surely `{}` is truthy as well?
```suggestion
                additionalProperties (assoc (s/optional-key s/Any) s/Any))
```

Reply:
Yep, you're right, missed that. Will push an update